### DISCUSSION TITLE
Discussion: Popup of the last changes with every new version

### DISCUSSION THREAD

**Discussion by DannyHatcher-Git** [RafaelGB/obsidian-db-folder#591] [Ideas]
### Present your request

Some of the other plugins have an update readme that pops up on new releases.

I just updated to 2.8.2 and there was no README.

Although I can come to the repo I would like a README to pop up in Obsidian telling me what is new and fixed.

### For which platform do you request this request??

Desktop

**Comment by RafaelGB** [comment]
the info of every update is avaliable on release notes.

I think a pop up for every new version is too intrusive.

The tag of the version is a direct link to those notes

**Comment by DannyHatcher-Git** [comment]
ah yes, I see what you mean now with the tag.

I had never noticed that before.

**Reply by RafaelGB** [reply]
Also we have a link called `changelog` with a custom page inside the documentation with even more info =P
